settings/src/main/resources/settings_schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
```

Comment:
https://github.com/networknt/json-schema-validator/ does not currently support the newest JSON schema version (`2020-12`). I think we should fix this as a contribution (issue here: https://github.com/networknt/json-schema-validator/issues/427) but right now it's not essential that we have it as far as I can see.